src/sherpa_ai/utils.py

```diff
@@ -325,3 +330,217 @@ def check_if_number_exist(result: str, source: str):
         message = f"Don't use the numbers {message} to answer the question instead stick to the numbers mentioned in the context."
         return {"number_exists": False, "messages": message}
     return {"number_exists": True, "messages": message}
+
+def string_comparison_with_jaccard_and_levenshtein(word1, word2, levenshtein_constant):
+    """
+        Calculate a combined similarity metric using Jaccard similarity and normalized Levenshtein distance.
+
+        Args:
+        - word1 (str): First input string.
+        - word2 (str): Second input string.
+        - levenshtein_constant (float): Weight for the Levenshtein distance in the combined metric.
```

Comment:
levenshtein_constant  => levenshtein_weight 

levenshtein_weight (float):  Weight between 0 and 1.0 for the Levenshtein distance in the combined metric.